ivy/functional/backends/numpy/statistical.py

```diff
@@ -253,3 +261,38 @@ def einsum(
 
 
 einsum.support_native_out = True
+
+
+def nanmedian(
+    a,
+    /,
+    *,
+    axis=None,
+    keepdims=False,
+    out=None,
+    dtype=None,
+    where=True,
+):
+    is_nan = ivy.isnan(a)
+    axis = tuple(axis) if isinstance(axis, list) else axis
+
+    if not any(is_nan):
+        if dtype:
+            a = ivy.astype(ivy.array(a), ivy.as_ivy_dtype(dtype))
+        ret = ivy.median(a, axis=axis, keepdims=keepdims, out=out)
+
+        if ivy.is_array(where):
+            ret = ivy.where(where, ret, ivy.default(out, ivy.zeros_like(ret)), out=out)
+    else:
+        a = [i for i in a if ivy.isnan(i) == False]
+
+        if dtype:
```

Comment:
comparison to False should be 'if cond is False:' or 'if not cond:'

Reply:
Done